Dockerfile

```diff
@@ -42,4 +42,3 @@ COPY root/ /
 
 # ports and volumes
 EXPOSE 80 443
-VOLUME /config
```

Comment:
I believe we need to do a `mkdir -p /config` to make sure the folder is present in case downstream stuff wants to write to it but it's not mounted as a volume.

Reply:
Although actually that's already on the Alpine baseimage so might not be necessary to do it here https://github.com/linuxserver/docker-baseimage-alpine/blob/master/Dockerfile#L76-L79

Reply:
We have https://github.com/linuxserver/docker-baseimage-alpine-nginx/blob/d7e627de1acf9f28a1110a17316c2e894f2cf4d5/root/etc/cont-init.d/11-folders#L4-L7 in this repo as well.